internal/cli/server/proto/server_grpc.pb.go

```diff
@@ -1,4 +1,8 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
```

Comment:
this code generation is it in Makefile or has been done by hands?

Reply:
I believe it is through Makefile using the command `make protoc`.